lib/sdk11/components/libraries/bootloader_dfu/bootloader.c

```diff
@@ -50,6 +50,8 @@ typedef enum
 static pstorage_handle_t        m_bootsettings_handle;  /**< Pstorage handle to use for registration and identifying the bootloader module on subsequent calls to the pstorage module for load and store of bootloader setting in flash. */
 static bootloader_status_t      m_update_status;        /**< Current update status for the bootloader module to ensure correct behaviour when updating settings and when update completes. */
 
+static bool m_cancel_timeout_on_usb;

```

Comment:
Please put a few text describing the variable just in case we forgot later on e.g

```
// Should we jump to application if USB is not mounted in specificed interval
```